### ISSUE TITLE
HttpOnly cookies or request headers?

### ISSUE THREAD

**Issue by luigifaticoso** [dart-flitter/flutter_webview_plugin#116]
Hello, 
getting the cookies with javascript doesn't let me get all the cookies as i need them for the authentication, is there any other way to get the cookies or to get the http request headers? 
Thank you

**Comment by UG** [comment]
I have same problemn getting http only session Id , generated by framework.
any how to access, response header parameter.

**Comment by aicheung** [comment]
Yes, I am also requesting the access of request and response headers in one of the event handlers too. We are doing SSO using the webview and the API token is in the header section in one of the AJAX call thus we need to access it otherwise SSO will not work.

**Comment by MichaelPriebe** [comment]
Need this.

**Comment by MrDragonn** [comment]
Need this.

**Comment by varpa89** [comment]
any news?

**Comment by loshine** [comment]
Need this too.

**Comment by charafau** [comment]
Ok, seems like we have some problems with cookies. This might be related - https://github.com/fluttercommunity/flutter_webview_plugin/pull/351 but I don't have anything to test it on so I need your help. If someone could contact me (on tweeter for example) and help with this issue that would be perfect. 

Thanks

**Comment by zhongmeizhi** [comment]
how to get cookie of httpOnly，怎么获取httpOnly的cookie，我也需要啊。

**Comment by irineusturza** [comment]
Hi, smae problem here. header == null in ajax's calls
